### PR TITLE
Adding page about the Markerview Plugin

### LINKED ISSUE #618: Add page/docs for MarkerView Plugin
MarkerView plugin is on the horizon https://github.com/mapbox/mapbox-plugins-android/pull/652#pullrequestreview-152989790

@mapbox/maps-android

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #618 by adding a page about the [Markerview Plugin for Android](https://github.com/mapbox/mapbox-plugins-android/tree/master/plugin-markerview)

### CODE REVIEW

**Comment by langsmith** [comment]
Ready for review @colleenmcginnis

**Comment by langsmith** [comment]
screencapture-localhost-8080-android-docs-plugins-overview-markerview-2018-11-07-13_37_53

File: src/pages/plugins/overview/markerview.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+	markerViewManager?.removeMarker(it)
+}
+`}
+/>
+}} 
+
+
+## Destroying the manager
+
+Don't forget to destroy your `MarkerViewManager` object! The destruction should happen in `onDestroy()` if you're using an activity. It should happen in the `onDestroyView()` method of a fragment.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/markerview.md]: `!` --> `.` 

😉

File: src/pages/plugins/overview/markerview.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+prependJs:
+  - |
+    import { MARKERVIEW_PLUGIN_VERSION } from '../../../constants';
+  - "import CodeLanguageToggle from '../../../components/code-language-toggle';"
+  - "import ToggleableCodeBlock from '../../../components/toggleable-code-block';"   
+---
+
+The MarkerView Plugin for Android provides a simplified way to add map markers that are Android views. 
+
+Please be aware that Android views are the less performant and less customizable option for adding icons to a map. The Mapbox Maps SDK's `SymbolLayer` is the recommended way to add icons. [View the Mapbox Annotation Plugin for Android](https://www.mapbox.com/android-docs/plugins/overview/annotation) for more information about using a `SymbolLayer`.
```
[the remark this answers is not in the document]
Comment (colleenmcginnis) [reply src/pages/plugins/overview/markerview.md]: Oh. Just gotta be in HTML rather than Markdown.

[the remark this answers is not in the document]
Comment (langsmith) [reply src/pages/plugins/overview/markerview.md]: Fixed

screen shot 2018-11-08 at 10 53 56 am

**Comment by langsmith** [comment]
Ok, I think we're good here @colleenmcginnis

**Pull request merged**